app/Entry.js

```diff
@@ -101,15 +101,55 @@ const SelfAssessmentStack = () => (
   </Stack.Navigator>
 );
 
+const moreTabScreenOptions = {
+  headerStyle: {
+    backgroundColor: Colors.primaryViolet,
+  },
+  headerTitleStyle: {
+    color: Colors.white,
+    textTransform: 'uppercase',
+  },
+  headerBackTitleVisible: false,
+  headerTintColor: Colors.white,
+};
+
 const MoreTabStack = () => (
-  <Stack.Navigator screenOptions={SCREEN_OPTIONS}>
-    <Stack.Screen name={Screens.Settings} component={SettingsScreen} />
-    <Stack.Screen name={Screens.About} component={AboutScreen} />
-    <Stack.Screen name={Screens.Licenses} component={LicensesScreen} />
-    <Stack.Screen name={Screens.FeatureFlags} component={FeatureFlagsScreen} />
-    <Stack.Screen name={Screens.Import} component={ImportScreen} />
-    <Stack.Screen name={Screens.ImportFromUrl} component={ImportFromUrl} />
-    <Stack.Screen name={Screens.ENDebugMenu} component={ENDebugMenu} />
+  <Stack.Navigator>
```

Comment:
Solid! Is it possible to add as the default options the `moreTabScreenOptions` and override the two that don't have the same settings with an empty set? Is probably not important but the diff might get smaller. 😁

Reply:
Totally! Good idea.